### PR TITLE
Added support for LFS clones

### PR DESCRIPTION
**Pull request by pieterclaerhout**
As per issue #67

### CODE REVIEW

File: bin/github-backup [hunk 1]
```diff
@@ -780,17 +787,26 @@ def fetch_repository(name,
             git_command = ['git', 'remote', 'set-url', 'origin', remote_url]
             logging_subprocess(git_command, None, cwd=local_dir)
 
-        git_command = ['git', 'fetch', '--all', '--force', '--tags', '--prune']
+        if lfs_clone:
+            git_command = ['git', 'lfs', 'fetch', '--all', '--force', '--tags', '--prune']
```
Comment (josegonzalez) [inline bin/github-backup]: Does this also get the non-lfs content?

Comment (pieterclaerhout) [reply bin/github-backup]: Yes, it does.

You do need to have ```git-lfs``` installed though.

https://git-lfs.github.com

Comment (josegonzalez) [reply bin/github-backup]: We should both document that and also toss in a check to ensure its installed if the `lfs` flag is used.

**Comment by josegonzalez** [comment]
Released in 0.14.0.

**Pull request merged**